roles/install-JDK-role/README.md

```diff
@@ -0,0 +1,38 @@
+Role Name
+=========
+
+A brief description of the role goes here.
```

Comment:
gentle reminder; `README` needs to be updated to reflect what this does.

Reply:
plz kindly update README , this content is autogenerated ;)

Reply:
working on it